src/centralstore/incremental/NodeManager.cpp

```diff
@@ -0,0 +1,254 @@
+/**
+Copyright 2020 JasminGraph Team
```

Comment:
Change `JasminGraph` to `JasmineGraph`

Reply:
Fixed in https://github.com/miyurud/jasminegraph/pull/115/commits/1d404d7d8fc79e4d908dcc39667a33b78beeaa26

Reply:
Done